Dockerfile

```diff
@@ -98,33 +104,14 @@ ENV INVENTREE_COMMIT_HASH="${commit_hash}"
 ENV INVENTREE_COMMIT_DATE="${commit_date}"
 
 # Copy source code
-COPY InvenTree ${INVENTREE_HOME}/InvenTree
-
-# Copy other key files
-COPY requirements.txt ${INVENTREE_HOME}/requirements.txt
-COPY tasks.py ${INVENTREE_HOME}/tasks.py
-COPY docker/gunicorn.conf.py ${INVENTREE_HOME}/gunicorn.conf.py
-COPY docker/init.sh ${INVENTREE_MNG_DIR}/init.sh
-
-# Need to be running from within this directory
-WORKDIR ${INVENTREE_MNG_DIR}
-
-# Drop to the inventree user for the production image
-#RUN adduser inventree
-#RUN chown -R inventree:inventree ${INVENTREE_HOME}
-#USER inventree
-
-# Install InvenTree packages
-RUN pip3 install --user --disable-pip-version-check -r ${INVENTREE_HOME}/requirements.txt
-
-# Server init entrypoint
-ENTRYPOINT ["/bin/bash", "./init.sh"]
+COPY InvenTree ./InvenTree
```

Comment:
Should we clean up what we ship in a next step? Ie. remove contrib things, assets and anything else that is not needed to run the app.

Reply:
I don't think there is much we can remove, we already only copy the InvenTree folder.